<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xVEquationDesignDlg</class>
 <widget class="QDialog" name="xVEquationDesignDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="pControlLab">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <property name="text">
          <string>controlled object</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="pControlledObjCB">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="xVPropTable" name="pSrcPropTable">
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="pMoveInTB">
       <property name="text">
        <string>add to equation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pMoveOutTB">
       <property name="text">
        <string>remove from equation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pSetAsResultTB">
       <property name="text">
        <string>set as result</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="pEqCapLab">
       <property name="text">
        <string>equation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="pEquationLEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="pCancelTB_2">
       <property name="text">
        <string>&amp;verify</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pCancelTB">
       <property name="text">
        <string>&amp;cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pApplyTB">
       <property name="text">
        <string>&amp;apply</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>xVPropTable</class>
   <extends>QTableWidget</extends>
   <header location="global">xvproptable.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>pControlledObjCB</tabstop>
  <tabstop>pSrcPropTable</tabstop>
  <tabstop>pMoveInTB</tabstop>
  <tabstop>pMoveOutTB</tabstop>
  <tabstop>pSetAsResultTB</tabstop>
  <tabstop>pEquationLEdit</tabstop>
  <tabstop>pCancelTB_2</tabstop>
  <tabstop>pCancelTB</tabstop>
  <tabstop>pApplyTB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pCancelTB</sender>
   <signal>clicked()</signal>
   <receiver>xVEquationDesignDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>459</x>
     <y>328</y>
    </hint>
    <hint type="destinationlabel">
     <x>345</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pApplyTB</sender>
   <signal>clicked()</signal>
   <receiver>xVEquationDesignDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>521</x>
     <y>324</y>
    </hint>
    <hint type="destinationlabel">
     <x>375</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
